python_modules/dagster/dagster/_utils/caching_instance_queryer.py

```diff
@@ -746,6 +766,8 @@ def get_asset_partitions_updated_after_cursor(
             asset_partitions (Optional[Sequence[AssetKeyPartitionKey]]): If supplied, will filter
                 the set of checked partitions to the given partitions.
             after_cursor (Optional[int]): The cursor after which to look for updates.
+            use_asset_versions (bool): If True, will use data versions to filter out asset
+                partitions which were materialized, but not updated after the cursor.
```

Comment:
when you say "but not updated", do you mean that they have the same data version as their previous materialization? if so, I think it would be clearest to spell that out fully